week-2-practise/level-3/5-indexOf.js

```diff
@@ -6,7 +6,8 @@
 // - return a new array with the replacement value inserted
 
 function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+  return newArr = (arr.slice(0, arr.indexOf(valueToReplace))
```

Comment:
Sorry, this doesn't work as expected! Please try again :)
Bare in mind my tips from above examples too